Gemfile

```diff
@@ -21,4 +21,9 @@ group :development do
   gem("byebug", require: false)
   gem("minitest-focus", require: false)
   gem("ruby-lsp", "~> 0.2.1")
+
+  gem("m")
```

Comment:
Is adding these dependencies via the gem file better practice now than using the gemspec dev dependencies, or is this being done to satisfy Sorbet?